### ISSUE TITLE
After converting, island owners are gone

### ISSUE THREAD

**Issue by Aricius2** [BentoBoxWorld/Converter#19]
**Description**
Not sure if this does belong here. But, after failed convertion, is original askyblock folder corrupted? We found (don´t know how the hell it happend), but on our 1.12.2 server running ASkyBlock we found BentoBox folder and from now, if we try to convert it to 1.16.5, all the island owners are gone. Any help? We did it multiple times succesfully. We checked backups and there was no BentoBox folder when conversion was succesful. Thanks (That server never went to 1.16.5)

**Comment by tastybento** [comment]
No, the AskyBlock folder is untouched. All the data should still be in the database. Do you have a backup of the AskyBlock folder just in case?

**Comment by Aricius2** [comment]
Sadly, we do but it is like a month old. We can´t use that. But even with that bentobox folder, ASkyBlock on 1.12.2 works just fine. It is just when converting, when we check someone, it just says player does not have a island. Tried multiple times, even without any other plugins. It worked like 6 times before. We used the same approach every time

**Comment by Aricius2** [comment]
Sorry. Think we got it. It looks like using older version of BentoBox  is working. Somehow newest version doesnt like converter. Will post a update if it was succesfull

**Comment by Aricius2** [comment]
Yup. BentoBox 1.16.1 was the problem. Did it with 1.16.0 as before and everything works fine.

**Comment by tastybento** [comment]
Did you receive any errors in the console when you used the newer version? I'd like to fix them.

**Comment by Aricius2** [comment]
Nope. No errors. Same as with older working version.
